news/2025/2025-06-05-osulazer-updates-june-5-2025.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "osu!(lazer) Updates: June 5, 2025"
+date: 2025-06-05 12:00:00 +0000
+---
+
+Experts say this might be THE osu!(lazer) update of all time.
+
+![](/wiki/shared/news/2025-06-05-osulazer-updates-june-5-2025/banner.jpg)
+
+You might have noticed that the rate at which news and YouTube videos has slowed down a bit, however the devs have not slowed down one bit, with several smaller updates being pushed out in April. That being said, this update is major, and we're excited to showcase it to you.
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/dQw4w9WgXcQ" frameborder="0" allowfullscreen></iframe>
+</div>
+
+### Gameplay
+
+- **The new song select is here for public testing.** To see the new design, simply input the simple *secret code* of... holding the "solo" button for a second before releasing. Very secretive, we know.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2025-06-05-song-select-v2.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+- **The leaderboard is now a skinnable element.** While it's not "skinnable" yet, this means you can manipulate its placement and size via the skin editor under the ruleset "HUD" layer.
+- **Add non-skinnable position indicator to multiplayer player.** With the update to the leaderboard, [peppy](https://osu.ppy.sh/users/2) wanted to make sure that players are still aware that they are in a multiplayer match, which is why the position indicator is now added to the game. Worry not, you can still hide it with the `Tab` key, but it will always show up during breaks.
```

Comment:
```suggestion
- **Add non-skinnable position indicator to multiplayer player.** With the update to the leaderboard, [peppy](https://osu.ppy.sh/users/2) wanted to make sure that players are still aware that they are in a multiplayer match, which is why the position indicator is now added to the game. Worry not, you can still hide it with the `Tab` key, but it will always show during breaks.
```

also, may be worth showcasing this with https://discord.com/channels/188630481301012481/1191872506420801546/1379884925565796372